ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/navigation-right/navigation-right.tsx

```diff
@@ -60,6 +61,7 @@ const Root = styled<Props, 'div'>('div')`
     width: auto;
     overflow: hidden;
     text-overflow: ellipsis;
+    border-left: solid 1px ${transparentize(0.8, '#ffffff')};
```

Comment:
Will this work for all themes?

Reply:
The only exception would be a totally white background on a custom defined theme. In that case we would want light gray. It's not intended to be very bold.